src/Core/Controllers/AdminController.php

```diff
@@ -321,6 +321,9 @@ protected function setupView($template)
         $user = Sentinel::getUser();
         $user->setDisabledActions($this->disabledActions);
 
+        $breadcrumbService = $this->app->make('Just\Shapeshifter\Services\BreadcrumbService');
```

Comment:
Lets use the PHP ::class caller method here. so: `Just\Shapeshifter\Services\BreadcrumbService:class` instead of a string.